apps/src/StudioApp.js

```diff
@@ -373,6 +373,10 @@ StudioApp.prototype.init = function(config) {
 
   if (this.isUsingBlockly()) {
     this.handleUsingBlockly_(config);
+  } else {
+    // handleUsingBlockly_ already does an onResize. We still want that goodness
+    // if we're not blockly
+    this.onResize();
```

Comment:
Considered that, but handleUsingBlockly_ does it in the middle (before an afterInject and before setStartBlocks) such that I would be worried about breaking things.